src/slate-editor/element.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import { RenderElementProps } from "slate-react";
+import { CustomElement, EFormat } from "../common/slate-types";
+
+function eltRenderAttrs(element: CustomElement) {
+  const attrs: any = {};
+
+  if ((element as any).align) {
+    attrs.textAlign = (element as any).align;
+  }
+
+  return attrs;
+}
+
+export const elementTypeMap: Record<string, string> = {
```

Comment:
This isn't great. It'd be nice to use the same class/object/structure for serialization, deserialization and rendering. Right now this is essentially a duplicate of the html tags in elementComponents below.